x-pack/plugins/timelines/public/components/hover_actions/actions/add_to_timeline.test.tsx

```diff
@@ -0,0 +1,373 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty } from '@elastic/eui';
+import { fireEvent } from '@testing-library/dom';
```

Comment:
Hi! A small nit 🙂. I think `fireEvent` can be imported from `@testing-library/react`